CHANGES.md

```diff
@@ -17,6 +17,7 @@ Release Notes.
 * Metrics combination API supports abandoning results.
 * Add a new concept "Event" and its implementations to collect events.
 * Add some defensive codes for NPE and bump up Kubernetes client version to expose exception stack trace.
+* Refactor StorageBuilder to remove the dependency of Map.
```

Comment:
I've been thinking whether we should include (nearly) all kinds of changes here, IMO the changes is targeted to our end users, who don't care about our code lever refactor at all, as well as some chores like CI settings, etc.

Reply:
I am not sure either. I just want to make this kind of change clear to all developer. Do you want me to remove this?

Reply:
Let me remove this.